pipelines/rh-push-to-registry-redhat-io/README.md

```diff
@@ -16,6 +16,10 @@ Tekton pipeline to release content to registry.redhat.io registry.
 | verify_ec_task_bundle | The location of the bundle containing the verify-enterprise-contract task | No | - |
 
 
+## Changes in 1.6.0
+* The publish-pyxis-repository task now sets source_container_image_enabled if `pushSourceContainer`
```

Comment:
I think this should first say that we now supply the new parameter. And then explain what it does as you do.

Reply:
updated